circle.yml

```diff
@@ -59,7 +59,7 @@ test:
         parallel: true
   post:
     - .circle/docker.sh build st2bundle
-    - .circle/docker.sh build st2actionrunner st2api st2auth st2exporter st2notifier st2resultstracker st2rulesengine st2sensorcontainer
+    - .circle/docker.sh build st2actionrunner st2api st2auth st2exporter st2notifier st2resultstracker st2rulesengine st2sensorcontainer st2garbagecollector
```

Comment:
Yeah, it's a new service which is part of  st2reactor package (for now).

I will have a look, we need to make sure all this gets documented :)